examples/knative/go/kn-go-tagging/function.yaml

```diff
@@ -0,0 +1,52 @@
+apiVersion: serving.knative.dev/v1
+kind: Service
+metadata:
+  name: kn-go-tagging
+  labels:
+    app: veba-ui
+spec:
+  template:
+    metadata:
+      annotations:
+        autoscaling.knative.dev/maxScale: "1"
+        autoscaling.knative.dev/minScale: "1"
+    spec:
+      containers:
+        - image: us.gcr.io/daisy-284300/veba/kn-go-tagging:1.0
+          imagePullPolicy: IfNotPresent
+          env:
+            - name: VCENTER_INSECURE
+              value: "false"
+            - name: VCENTER_URL
+              value: "https://FILL-ME-IN"
+            # - name: VCENTER_SECRET_PATH
+            #   value: "/var/bindings/vsphere"
+            - name: DEBUG
+              value: "true"
+            - name: TAG_NAME
+              value: "FILL-ME-IN"
+            - name: TAG_ACTION
+              value: "FILL-ME-IN" # can be "attach" or "detach"
+          volumeMounts:
+            - name: credentials
+              mountPath: /var/bindings/vsphere
+              readOnly: true
+      volumes:
+        - name: credentials
+          secret:
+            secretName: vsphere-credentials
+---
+apiVersion: eventing.knative.dev/v1
+kind: Trigger
+metadata:
+  name: kn-go-tagging-trigger
+spec:
+  broker: default
+  filter:
+    attributes:
+      subject: VmPoweredOffEvent
```

Comment:
suggestion: Would be great to add: `type: com.vmware.event.router/event` here to make the filter more specific and avoid getting duplicate events if the kn-go-echo function is deployed.

Reply:
Done.